gyp/pylib/gyp/common.py

```diff
@@ -429,6 +429,8 @@ def GetFlavor(params):
     return 'netbsd'
   if sys.platform.startswith('aix'):
     return 'aix'
+  if sys.platform.startswith('os390'):
+    return 'os390'
```

Comment:
if the platform is name `zos` why use `os390`?

Reply:
Pretty sure that's what `uname -m` prints on z/OS machines.

Reply:
Ack. So that's the input (L432), I'm wondering is we should keep it as an the internal alias.
We could do something similar to:
https://github.com/nodejs/node-gyp/blob/2ed26fbf2bf7257a3e21a12da8822ccd4e79e9de/gyp/pylib/gyp/common.py#L422-L423
if that's more congruent with the IBM's future plans.

Reply:
A lot of z/OS projects built using gyp already expect 'os390'. So that would require a major change across multiple z/OS projects.

Reply:
I meant the other way around, i.e.:
```python
   if sys.platform.startswith('os390'): 
     return 'zos'
```
since the return `os390` (a.k.a. `flavor`) is only used as an internal alias to map to https://github.com/nodejs/node-gyp/blob/2ed26fbf2bf7257a3e21a12da8822ccd4e79e9de/gyp/pylib/gyp/generator/make.py#L2060

But in anycase, You guys know best. I'm just asking...

Reply:
@jBarz this is just what we refer to it as inside the node build scripts, so it shouldn't affect other projects.

However I think `os390` is the standard "technical" name for the ARCH, like `x86_64` is for Intel, so it's probably be less surprising to stick with that.

Reply:
@refack apologies, you're right that this is only internal alias.
@gibfahn is right though, 'os390' is a standard technical name for z/OS arch. So I would prefer to leave it as 'os390' if that is ok with you